src/Components/CreateImageWizard/steps/Review/ReviewStep.tsx

```diff
@@ -351,7 +364,8 @@ const Review = ({ snapshottingEnabled }: { snapshottingEnabled: boolean }) => {
           <TimezoneList />
         </ExpandableSection>
       )}
-      {isLocaleEnabled && (
+      {((isLocaleEnabled && languages && languages.length > 0) ||
+        (isLocaleEnabled && keyboard && keyboard.length > 0)) && (
```

Comment:
Similarly here, first check for the flag which doesn't combine with anything and then check whether there are any values:
```suggestion
      {isLocaleEnabled &&
        ((languages && languages.length > 0) ||
          (keyboard && keyboard.length > 0)) && (
```